projects/ad7616-sdz/src/ad7616_sdz.c

```diff
@@ -70,9 +74,29 @@ struct spi_engine_offload_init_param spi_engine_offload_init_param = {
 struct spi_engine_init_param spi_eng_init_param  = {
 	.ref_clk_hz = 100000000,
 	.type = SPI_ENGINE,
-	.spi_engine_baseaddr = AD7616_CORE_BASEADDR,
+	.spi_engine_baseaddr = AD7616_SPI_ENGINE_BASEADDR,
 	.cs_delay = 1,
-	.data_width = 8,
+	.data_width = 16,
+};
+
+struct axi_clkgen_init clkgen_init = {
+	.name = "rx_clkgen",
+	.base = AD7616_RX_CLKGEN_BASEADDR,
+	.parent_rate = 100000000,
+};
+
+struct axi_pwm_init_param axi_pwm_init_params = {
+	.base_addr = AXI_PWMGEN_BASEADDR,
+	.ref_clock_Hz = 100000000,
+	//.channel = 0,
```

Comment:
in general we do not leave commented code in commits, if this is not needed simply remove it.